js/src/util/backdrop.js

```diff
@@ -138,6 +142,12 @@ class Backdrop extends Config {
       execute(this._config.clickCallback)
     })
 
+    if (this._config.keydownCallback) {
+      EventHandler.on(element.parentElement, EVENT_KEYDOWN, event => {
+        execute(this._config.keydownCallback(event))
```

Comment:
```suggestion
        execute(this._config.keydownCallback, [event])
```
suppose is better this way